docs/auth/README.adoc

```diff
@@ -0,0 +1,116 @@
+= AeroGear Services Auth SDK
+
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+== Usage
+
+To use the Auth SDK you'll first need to:
+
+* Have a Keycloak instance. See the <<Setting up Keycloak>> section.
+* A configuration file added to the app directory. See the <<Configuration File>> section.
+
+=== Setting up Keycloak
+
+==== Using OpenShift
+
+* If you do not have mobile services enabled in your openshift cluster follow this link:https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/local-setup.adoc[Local Setup] guide.
+* Navigate to your Openshift cluster and in the Service Catalog search for the Keycloak service.
+* Click on the Keycloak service and you will be prompted to fill in details about your app.  For now you can leave these as they are.  Navigate through the setup and click Create.
+This will provision the Keycloak service in the project you specified and create a public Client to be used with an app along with a bearer-only client.
+See link:http://www.keycloak.org/docs/latest/server_admin/index.html#oidc-clients[Keycloak OIDC client documentation].
+
+==== Standalone
+
+To setup standalone Keycloak follow Keycloak's guide link:/https://github.com/keycloak/keycloak/blob/master/README.md[here].
```

Comment:
are the above section the same as in https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/modules/getting-started/pages/auth.adoc?

Reply:
Yes, all 3 Auth SDK's have the some repeated text at the top as there is no shared docs space at the moment from what I can see. :(

Reply:
@finp any suggestions? are you working on a shared docs repo yet?

Reply:
any generic topics should be pushed to mobile-docs, with the following in mind:
* not yet, I need to merge my antora config changes
* not applicable for just sections
Let me get back to you on this, as it's an important subject and needs a structured approach